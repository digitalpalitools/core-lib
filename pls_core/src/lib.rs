#[macro_use]
extern crate lazy_static;
extern crate serde_json;

pub mod alphabet;
pub mod inflections;
