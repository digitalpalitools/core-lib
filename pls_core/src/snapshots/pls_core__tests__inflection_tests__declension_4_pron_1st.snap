---
source: pls_core/src/lib.rs
expression: html

---
<div class="pls-inflection-root">
  <header class="pls-inflection-header">
    <summary class="pls-inflection-summary">^maṃ$ &ndash; "ahaṃ pron 1st" (irregular)</summary>
    </header>
<table class="pls-inflection-table pls-inflection-type-declension">
  <thead>
    <tr>
      <td><span class="pls-inflection-table-title">ahaṃ pron 1st</span></td>
      <td><span class="pls-inflection-col-header">1st sg</span></td>
      <td><span class="pls-inflection-col-header">1st pl</span></td>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>
        <p><span class="pls-inflection-row-header">nom</span></p>
      </td>
      <td>
        <div class="pls-inflection-inflected-word">^ahaṃ$</div>
        </td>
      <td>
        <div class="pls-inflection-inflected-word">^amhe$</div>
        <div class="pls-inflection-inflected-word">^mayaṃ$</div>
        </td>
    </tr>
    <tr>
      <td>
        <p><span class="pls-inflection-row-header">acc</span></p>
      </td>
      <td>
        <div class="pls-inflection-inflected-word">^mamaṃ$</div>
        <div class="pls-inflection-inflected-word">^maṃ$</div>
        <div class="pls-inflection-inflected-word">^me$</div>
        </td>
      <td>
        <div class="pls-inflection-inflected-word">^amhākaṃ$</div>
        <div class="pls-inflection-inflected-word">^amhe$</div>
        <div class="pls-inflection-inflected-word">^no$</div>
        </td>
    </tr>
    <tr>
      <td>
        <p><span class="pls-inflection-row-header">instr</span></p>
      </td>
      <td>
        <div class="pls-inflection-inflected-word">^mayā$</div>
        <div class="pls-inflection-inflected-word">^me$</div>
        </td>
      <td>
        <div class="pls-inflection-inflected-word">^amhehi$</div>
        <div class="pls-inflection-inflected-word">^no$</div>
        </td>
    </tr>
    <tr>
      <td>
        <p><span class="pls-inflection-row-header">dat</span></p>
      </td>
      <td>
        <div class="pls-inflection-inflected-word">^mama$</div>
        <div class="pls-inflection-inflected-word">^mamaṃ$</div>
        <div class="pls-inflection-inflected-word">^mayha$</div>
        <div class="pls-inflection-inflected-word">^mayhaṃ$</div>
        <div class="pls-inflection-inflected-word">^me$</div>
        </td>
      <td>
        <div class="pls-inflection-inflected-word">^amhaṃ$</div>
        <div class="pls-inflection-inflected-word">^amhākaṃ$</div>
        <div class="pls-inflection-inflected-word">^no$</div>
        </td>
    </tr>
    <tr>
      <td>
        <p><span class="pls-inflection-row-header">abl</span></p>
      </td>
      <td>
        <div class="pls-inflection-inflected-word">^mayā$</div>
        </td>
      <td>
        <div class="pls-inflection-inflected-word">^amhehi$</div>
        </td>
    </tr>
    <tr>
      <td>
        <p><span class="pls-inflection-row-header">gen</span></p>
      </td>
      <td>
        <div class="pls-inflection-inflected-word">^mama$</div>
        <div class="pls-inflection-inflected-word">^mamaṃ$</div>
        <div class="pls-inflection-inflected-word">^mayha$</div>
        <div class="pls-inflection-inflected-word">^mayhaṃ$</div>
        <div class="pls-inflection-inflected-word">^me$</div>
        </td>
      <td>
        <div class="pls-inflection-inflected-word">^amhaṃ$</div>
        <div class="pls-inflection-inflected-word">^amhākaṃ$</div>
        <div class="pls-inflection-inflected-word">^no$</div>
        </td>
    </tr>
    <tr>
      <td>
        <p><span class="pls-inflection-row-header">loc</span></p>
      </td>
      <td>
        <div class="pls-inflection-inflected-word">^mayi$</div>
        </td>
      <td>
        <div class="pls-inflection-inflected-word">^amhesu$</div>
        </td>
    </tr>
    <tr>
      <td><span class="pls-inflection-row-header">in comps</span></td>
      <td colspan="2">
        </td>
    </tr>
  </tbody>
</table>

  <footer class="pls-inflection-footer">
    <a class="pls-inflection-feedback-link" target="_blank" href="https://docs.google.com/forms/d/e/1FAIpQLSeoxZiqvIWadaLeuXF4f44NCqEn49-B8KNbSvNer5jxgRYdtQ/viewform?entry.267696921=test case&entry.1433863141=v0.1">
      spot a mistake? something missing? fix it here!
    </a>
  </footer>
</div>

